ANS_JS_template.tex

```diff
@@ -93,92 +93,92 @@
 
-First, we will provide a literature review covering both nuclear and non nuclear hybrid energy systems.  We will then discuss the ongoing research for modeling a nuclear hybrid energy system to evaluate the important characteristics to include in a model. Third, we will describe nuclear fuel cycle simulators. Then we will analyze the possibilities of using a nuclear fuel cycle simulator to model a nuclear hybrid energy system. The goal of the research presented in this paper is to determine the benefits and drawbacks of using a nuclear fuel cycle simulator (NFCS) to model nuclear renewable hybrid energy systems (NRHES).  Accessibility through using an open source platform and a popular programming language are found to be the primary benefits of using a NFCS. As the CYCLUS fuel cycle simulator is the only open source NFCS, this paper will culminate in the possible applications of CYCLUS for modeling NRHES.
+First, we will provide a literature review covering both nuclear and non nuclear hybrid energy systems.  We will then discuss the ongoing research for modeling a NRHES to evaluate the important characteristics to include in a software model. Third, we will describe nuclear fuel cycle simulators (NFCS). Then we will analyze the similarities between the functionality needed to model a NRHES and a NFCS. While there are differences between the two systems, most obviously time scale and the need for a dynamic feedback response in a NRHES, the two physical systems share some common characteristics. NFCSs are at a more mature state of development, with some capacity to inform future development of NRHES software models.
 
```

Comment:
Updated to demonstrate that the focus is no long on CYCLUS, but instead on how NRHES can benefit from the development of NFCS.